test/shared-class.test.js

```diff
@@ -198,6 +198,25 @@ describe('SharedClass', function() {
     });
   });
 
+  describe('sharedClass.findMethodByName()', function() {
+    var sc;
+    var sm;
+    it('finds sharedMethod by prototype name', function() {
+      sc = new SharedClass('SomeClass', SomeClass);
+      var PROTOTYPE_METHOD_NAME = 'SomeClass.prototype.testMethod';
+      sm = sc.defineMethod(PROTOTYPE_METHOD_NAME);
+      assert(sc.findMethodByName('SomeClass.prototype.testMethod') === sm);
+    });
+    it('find sharedMethod by name', function() {
```

Comment:
Also modify the test name to communicate that this test is checking static method.